packages/cdk8s-cli/src/cli/cmds/import.ts

```diff
@@ -18,6 +18,7 @@ class Command implements yargs.CommandModule {
     .example('cdk8s import k8s@1.13.0', 'Imports a specific version of the Kubernetes API')
     .example('cdk8s import jenkins.io_jenkins_crd.yaml', 'Imports constructs for the Jenkins custom resource definition from a file')
     .example('cdk8s import mattermost:=mattermost_crd.yaml', 'Imports constructs for the mattermost cluster custom resource definition using a custom module name')
+    .example('cdk8s import crossplane/crossplane@0.14.0', 'Imports constructs for a GitHub repo using doc.crds.dev')
```

Comment:
Should we change to `github://crossplane/crossplane@0.14.0` (or `gh://` or `gh:`). WDYT?

Add to line 16:

```
import spec with the syntax [NAME:=]SPEC where NAME is an optional module name and supported SPEC are: k8s, crd.yaml, https://domain/crd.yaml, github://account/repo[@VERSION]
```

Reply:
github:// sounds good. Will update.

Reply:
/cc @hasheddan WDYT? ☝️ 

I'm leaning towards the slightly shorter `cdk8s import github:crossplane/crossplane@0.14.0`

Reply:
`github:` is find by me

Reply:
`github:` SGTM

Reply:
fixed